drivers/wifi/nrf700x/zephyr/inc/zephyr_fmac_main.h

```diff
@@ -78,6 +78,13 @@ struct wifi_nrf_ctx_zep {
 	struct rpu_conf_params conf_params;
 #endif /* CONFIG_NRF700X_WIFI_UTIL */
 #endif /* CONFIG_NRF700X_RADIO_TEST */
+	u8 *extended_capa, *extended_capa_mask;
```

Comment:
Use `unsigned char` instead of `u8` to be consistent with all other elements of similar type in the structure.